conf/broker.conf

```diff
@@ -411,6 +411,11 @@ dispatcherReadFailureBackoffMandatoryStopTimeInMs=0
 # Precise dispathcer flow control according to history message number of each entry
 preciseDispatcherFlowControl=false
 
+# Class name of Pluggable entry filter that can decide whether the entry needs to be filtered
+# You can use this class to decide which entries can be sent to consumers.
+# Multiple classes need to be separated by commas.
+entryFilterClassNames=
```

Comment:
Do we need a directory to maintain all the filters? otherwise, we need to copy the filter jar to the lib dir.

Reply:
Another point is we should load the class by a separate classloader that extends from the pulsar classloader?